### DISCUSSION TITLE
Paid Support

### DISCUSSION THREAD

**Discussion by jfm-so** [rtl-airband/RTLSDR-Airband#395] [General]
Hi there - I work with a wildfire/natural disaster mitigation non-profit here in California, and we're looking to implement RTLSDR-Airband as part of upgrading from our antique analog radio stack. 

I apologize if this isn't the right place, but I couldn't find an email directly for charlie-foxtrot. 

I would love to get some personal one-on-one support with initial configuration and have a personal contact for troubleshooting on an as-needed basis. 

While as a small non-profit, we have a limited budget, I have no problem paying for your time at a reasonable invoice amount.

Johnathan Martin (jm@LAIT911.com)

**Comment by jfm-so** [comment]
@charlie-foxtrot :)

**Comment by charlie-foxtrot** [comment]
I'm not in a position to do 1:1 consulting but am curious what you guys are doing, will reach out via email.
